Backend/Models/Configurations/MedicamentConfiguration.cs

```diff
@@ -12,6 +12,10 @@ public void Configure(EntityTypeBuilder<Medicament> builder)
             builder
                 .HasOne(m => m.PharmaceuticalForm)
                 .WithMany(pf => pf.Medicaments);
+
+            builder
+                .HasMany(r => r.RequiredMedicamentAmounts)
```

Comment:
Variables in lambda expression should correspond with the type of object we inferring. Since these LINQ queries are quite short, we use one letter naming, which corresponds to class name. For example `m => m.PharmaceuticalForm`, where `m` type is `Medicament`. This naming pattern should be applied in other queries as well for consistency.

Reply:
Fixed